src/test/java/models/DefectResponse.java

```diff
@@ -0,0 +1,24 @@
+package models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DefectResponse {
+    @SerializedName("id")
+    private int id;
+    @SerializedName("title")
+    private String title;
+    @SerializedName("actual_result")
+    private String actual_result;
```

Comment:
Please use camelCase for variables names in java according to java code convention. Since you're using @SerializedName annotation to chnage property name to snake_case for api tests